server/src/payment/payment.service.ts

```diff
@@ -0,0 +1,84 @@
+import { Injectable } from '@nestjs/common';
+import { PrismaService } from '../prisma/prisma.service';
+import {
+  UpdatePaymentsStatusPerDateDto,
+  UpdatePaymentsStatusPerMonthDto,
+} from './dto';
+
+@Injectable()
+export class PaymentService {
+  constructor(private prisma: PrismaService) {}
+
+  async updatePaymentsStatusPerDate(dto: UpdatePaymentsStatusPerDateDto) {
```

Comment:
Add validation for the user belongs to the house same as perMonth!

Reply:
Fixed!
https://github.com/HAK2024/fairy-share/pull/102/commits/a1fc71f919bcd609119883c6483d1e4432799dc3